test/SlowTests/Server/Documents/TimeSeries/TimeSeriesTombstoneCleaner.cs

```diff
@@ -329,8 +329,12 @@ public async Task CleanTimeSeriesTombstonesInTheClusterWithOnlyFullBackup()
                         var c2 = storage.DocumentsStorage.TimeSeriesStorage.GetNumberOfTimeSeriesDeletedRanges(context);
                         var c3 = storage.DocumentsStorage.TimeSeriesStorage.GetNumberOfTimeSeriesPendingDeletionSegments(context);
 
-
-                        Assert.Equal(750, c2);
+                        // note that for most cases 'c2' should be equal to 750
+                        // but there are rare scenarios that the requests are sent twice. 
+                        // for example, in case the connection was lost.
+                        // in this case, we may get 1500 value which is fine,
```

Comment:
We need to maintain the consistency & integrity of the system. 
For example, we have 3 nodes in a cluster: A, B, C. 
Assuming that we got to a specific time which only A and B include TS _"HeartRates"_ - but C gets a deletion range request for _"HeartRates"_:
-> If we don't add the `TimeSeriesDeletedRanges` in C (because we are not removing anything, this TS doesn't exist for C) - the `TimeSeriesDeletedRanges` will not send to A and B; so when all nodes' data is fixed (same data in all nodes), the deletion request got lost. 

@karmeli87 is this correct?